o-markitdigital-equity-highlight/main.scss

```diff
@@ -17,9 +25,11 @@ $o-card-is-silent: false;
         @include oColorsFor(page);
         font-family: oFontsGetFontFamilyWithFallbacks(FinancierDisplayWeb);
         width: 260px;
-        height: 216px;
+        height: 200px;
 
-        div {
+        div, 
```

Comment:
These styles need to be scoped to some kind of parent selector for the component or you'll be setting global styles for every div, span and a on the page

Reply:
This was required to be applied to all the div, span and a on this app, so I have written this in such a manner wrapped in .o-equity-highlight-app { }